lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -444,10 +444,10 @@ private function getTableDiffForAlterForeignKey(ForeignKeyConstraint $foreignKey
     private function parseColumnCollationFromSQL($column, $sql)
     {
         if (preg_match(
-            '{(?:'.preg_quote($column).'|'.preg_quote($this->_platform->quoteSingleIdentifier($column)).')
+            '/(?:'.$this->pregQuote($column).'|'.$this->pregQuote($this->_platform->quoteSingleIdentifier($column)).')
```

Comment:
This is what I told you about CS violations: https://travis-ci.org/doctrine/dbal/jobs/285994030#L2398 can you please fix it?

Reply:
The violation was there already in the first place, but I'll fix it ...